dev-qt/qtcore/metadata.xml

```diff
@@ -6,6 +6,7 @@
 		<name>Gentoo Qt Project</name>
 	</maintainer>
 	<use>
+		<flag name="old-kernel">disable new syscalls for Linux kernel >= 3.17</flag>
```

Comment:
description is confusing, it sounds like the flag should be enabled for kernels >= 3.17, while the opposite is true.

Also, start with a capital letter.

Reply:
How about `Disable syscalls not available on Linux kernels older than 3.17`

Reply:
sounds good to me